src/Files.App/ValueConverters/StringToBrushConverter.cs

```diff
@@ -1,21 +1,18 @@
-using Files.App.Helpers;
-using Files.Backend.Models.Coloring;
+﻿using CommunityToolkit.WinUI.Helpers;
 using Microsoft.UI.Xaml.Data;
 using Microsoft.UI.Xaml.Media;
 using System;
 
 namespace Files.App.ValueConverters
 {
-	public sealed class ColorModelToColorConverter : IValueConverter
+	internal sealed class StringToBrushConverter : IValueConverter
```

Comment:
It seems we have two converters for this, (the other one is used in the appearance settings page). Can you see if they can be merged?

Reply:
Done!